.github/workflows/az-e2e.yaml

```diff
@@ -0,0 +1,34 @@
+name: azure e2e tests
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+jobs:
+  e2e:
+    name: operator azure e2e tests
+    runs-on: az-ubuntu-2204
+
+    strategy:
+      matrix:
+        runtimeclass: ["kata-qemu", "kata-clh"]
```

Comment:
I'm not sure if makes sense to run operator test for each runtimeClass. But let's leave as is for now.